enterprise/server/scheduling/scheduler_server/scheduler_server.go

```diff
@@ -280,6 +280,16 @@ func (h *executorHandle) setRegistration(r *scpb.ExecutionNode) {
 	h.registration = r
 }
 
+func truncateDuration(d, min, max time.Duration) time.Duration {
```

Comment:
nit: applying a min/max is usually called clamping - truncating is usually rounding to a lower precision

```suggestion
func clampDuration(d, min, max time.Duration) time.Duration {
```

Reply:
Thank you! I knew this was wrong but it was late and I couldn't think of the right word lol.